src/main/java/com/rcv/Tabulator.java

```diff
@@ -358,8 +358,13 @@ private void setWinningThreshold(Map<String, BigDecimal> currentRoundCandidateTo
       currentRoundTotalVotes = currentRoundTotalVotes.add(numVotes);
     }
 
-    // divisor for threshold is num winners + 1
-    BigDecimal divisor = new BigDecimal(config.getNumberOfWinners() + 1);
+    // divisor for threshold is num winners + 1 (unless archaic Hare quota option is enabled, in
```

Comment:
> archaic

ooo I hope they got aloe for that burn!

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/2e29187e-812d-4803-bdec-9e49ef28a2d5-0878c302-d67c-4bb7-80ba-b554896c3f48-v1.png?transparent=1&palette=1&width=246)